HikVisionSource/src/main/java/io/vantiq/extsrc/HikVisionSource/ImageUtil.java

```diff
@@ -0,0 +1,270 @@
+/*
+ * Copyright (c) 2020 Vantiq, Inc.
+ *
+ * All rights reserved.
+ *
+ * SPDX: MIT
+ */
+package io.vantiq.extsrc.HikVisionSource;
+
+//import edu.ml.tensorflow.model.BoxPosition;
+//import edu.ml.tensorflow.model.Recognition;
+import io.vantiq.client.BaseResponseHandler;
+import io.vantiq.client.ResponseHandler;
+import io.vantiq.client.Vantiq;
+import io.vantiq.client.VantiqError;
+import okhttp3.Response;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Util class for image processing.
+ */
+public class ImageUtil {
+    private final static Logger LOGGER = LoggerFactory.getLogger(ImageUtil.class);
+    public String outputDir = null; // Added to remember the output dir for each instance
+    public Vantiq vantiq = null; // Added to allow image saving with VANTIQ
+    public String sourceName = null;
+    public Boolean saveImage;
+    public int frameSize;
+    public Boolean queryResize = false;
+    public int longEdge = 0;
+    public Boolean uploadAsImage = false;
+
+    // Used to upload image to VANTIQ as VANTIQ Image
+    final static String IMAGE_RESOURCE_PATH = "/resources/images";
+
+    /**
```

Comment:
please remove all this commented-out code & comments.

Reply:
done